backend/app_fastapi_v2.py

```
@@ -217,6 +218,41 @@ async def get_user_details(current_user: dict = Depends(get_current_user)):
+            cancel_url=request.cancel_url,
+            metadata= {
+                'customer_name': f"{current_user.get('given_name', '')} {current_user.get('family_name', '')}".strip(),
+                'customer_email': current_user.get("email", ""),
+                'customer_id' : current_user["id"],
+                'product_id' : request.price_id
+            },
+        )
+        # Return the session ID
+        return {"session_id": session.id}
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)